### ISSUE TITLE
onnx prints output instead of saving

### ISSUE THREAD

**Issue by AlexMuresan** [onnx/onnx#2201]
When trying to export a model using torch.onnx.export() there is no file saved, instead I get a long stream of outputs in the terminal.

This is how I try to save:

    dummy_input = torch.randn(config.BATCH_SIZE, 1, 224, 224,
                                  requires_grad=True)
    input_names = ["actual_input_1"] + ["learned_%d" % i for i in range(16)]
    output_names = ["output1"]

    torch.onnx.export(model, dummy_input, os.path.join(path, "model.onnx"),
                      verbose=False, input_names=input_names,
                      output_names=output_names)

This is my model:

    class Flatten(nn.Module):
        def forward(self, input):
            return input.view(input.size(0), -1)

    class Model(nn.Module):
        def __init__(self, nn_w=1, in_h=28, in_w=28):
            super(Model, self).__init__()
            self.features = nn.Sequential(
                nn.Conv2d(1, nn_w*2, kernel_size=3, stride=2, padding=0),
                nn.BatchNorm2d((nn_w*2)),
                nn.ReLU(),

                nn.Conv2d(nn_w*2, nn_w*3, kernel_size=3, stride=1, padding=0),
                nn.BatchNorm2d((nn_w*3)),
                nn.ReLU(),

                nn.Conv2d(nn_w*3, nn_w*4, kernel_size=3, stride=1, padding=0),
                nn.BatchNorm2d((nn_w*4)),
                nn.ReLU(),

                nn.Conv2d(nn_w*4, nn_w*5, kernel_size=3, stride=1, padding=0),
                nn.BatchNorm2d((nn_w*5)),
                nn.ReLU(),

                nn.Conv2d(nn_w*5, nn_w*6, kernel_size=3, stride=1, padding=0),
                nn.BatchNorm2d((nn_w*6)),
                nn.ReLU(),

                nn.Conv2d(nn_w*6, 10, kernel_size=1, stride=1, padding=0),
                nn.AvgPool2d(kernel_size=5, stride=5),

                Flatten(),
                nn.Sigmoid(),
            )

        def forward(self, x):
            return self.features(x)

I'm expecting to find a file in the given path named 'model.onnx'

Actual output is 9000+ lines of this:

    2.7782e-11  2.1268e-10  7.9657e-10  5.8317e-10  1.2265e-13  6.5009e-12        
    8673e-14  1.0848e-08  5.0007e-14  7.0868e-10  3.0846e-14  2.7779e-12                  
    6577e-16  3.0803e-09  4.0009e-08  3.3402e-06  2.5615e-08  4.1427e-08

**Comment by AlexMuresan** [comment]
The issue seems to be in the torch code rather than onnx. More specifically it seems that when calling torch.onnx.export() on a model wrapped in DataParallel this issue will happen.

The following piece of code inside torch.onnx._export() seems to be the culprit:
    if isinstance(model, torch.nn.DataParallel):
        model = model.module

When the check is done outside of the torch library the model is properly striped from DataParallel and saved as an onnx file.

**Comment by prasanthpul** [comment]
Thanks @AlexMuresan. Since the issue is in PyTorch, please file in that github so the right folks can fix. @spandantiwari and @houseroad as FYI

**Comment by Tekno-H** [comment]
> The issue seems to be in the torch code rather than onnx. More specifically it seems that when calling _torch.onnx.export()_ on a model wrapped in DataParallel this issue will happen.
> 
> The following piece of code inside _torch.onnx._export()_ seems to be the culprit:
> 
> ```
> if isinstance(model, torch.nn.DataParallel):         
>     model = model.module
> ```
> 
> When the check is done outside of the torch library the model is properly striped from DataParallel and saved as an onnx file.

Hello, I found myself stuck at in a similar issue.
Could you give me any leads to help me solve this problem?
